scripts/helpers/time_series_helpers.py

```diff
@@ -0,0 +1,235 @@
+        order (int, int, int)
+        seasonal_order (int, int, int, int)
+        steps (int, str, datetime): Number of steps to forecast
+
+    Returns:
+        model_forecast (Series) of forecast values of length
+    """
+    # train Sarimax model with all data
+    model_sarimax = SARIMAX(endog=train.y,
+                            exog=exog,
+                            order=order,
+                            seasonal_order=seasonal_order).fit()
+
+    # make a forecast with Sarimax model trained on all data
+    model_forecast = model_sarimax.forecast(steps=steps)
+    return model_forecast
+
+
+def reshape_time_series_data(pdf, date_col, var_cols):
+    # cast date column and add as index
+    pdf = pdf[[date_col] + var_cols]
+    pdf = pdf.rename(columns={date_col: 'ds'})
+    pdf.ds = ps.to_datetime(pdf.ds, format='%d/%m/%Y')
+    pdf = pdf.set_index('ds').sort_index()
+    if len(var_cols) == 1:
+        pdf = pdf.rename(columns={var_cols[0]: 'y'})
+    elif len(var_cols) > 1:
+        for v_col in var_cols:
+            counter = 0
+            pdf = pdf.rename(columns={v_col: f'y_{counter}'})
+            pdf = pdf.astype({f'y_{counter}': float})
+            counter += 1
+        pdf = pdf.rename(columns={'y_0': 'y'})
+    return pdf
+
+
+def get_seasonal_decompose_plot(x, model, period, plot=False, fname=None):
+    """
+
+    Args:
+        fname ():
+        plot ():
+        df ():
+        model ():
+        period ():
+
+    Returns:
+
+    """
+    decompose = seasonal_decompose(x=x.dropna(), model=model, period=period)
+    trend = decompose.trend
+    seasonal = decompose.seasonal
+    residual = decompose.resid
+    if plot:
+        label_loc = 'upper left'
+        fig, axes = plt.subplots(4, 1, sharex=True, sharey=False)
+        fig.set_figheight(10)
+        fig.set_figwidth(15)
+        fig.suptitle('Decomposition of time series data')
+        axes[0].plot(x, label='Original')
+        axes[0].legend(loc=label_loc)
+        axes[1].plot(trend, label='Trend')
+        axes[1].legend(loc=label_loc)
+        axes[2].plot(seasonal, label='Cyclic')
+        axes[2].legend(loc=label_loc)
+        axes[3].plot(residual, label='Residuals')
+        axes[3].legend(loc=label_loc)
+        if fname:
+            plt.tight_layout()
+            plt.savefig(fname=fname)
+        plt.show()
+    return trend, seasonal, residual
+
+
+def plot_time_series_data(x, var_dict, title, xlabel, ylabel, fname=None):
+    """
+
+    Args:
+        x ():
+        var_dict (): dict containing label and variable names e.g. {'Number people': 'y'}
+        title ():
+        xlabel ():
+        ylabel ():
+
+    Returns:
+
+    """
+    fig = plt.subplots(figsize=(10, 8))
+    for k, v in var_dict.items():
+        plt.plot(x.index, x[v], label=k)
+    plt.title(title)
+    plt.xlabel(xlabel)
+    plt.ylabel(ylabel)
+    plt.legend()
+    if fname:
+        plt.tight_layout()
+        plt.savefig(fname=fname)
+    plt.show()
```

Comment:
I'm not sure you want this in a glue job, return the plot object then someone can show it if they want / are using a notebook?